pkg/webhook/cspc_operations.go

```diff
@@ -282,9 +237,9 @@ func (bdr *BlockDeviceReplacement) IsBDReplacementValid(newRG, oldRG *cstor.Raid
 
 // validateNewBDCapacity returns error only when new block device has less capacity
 // than existing block device
-func (bdr *BlockDeviceReplacement) validateNewBDCapacity(newRG, oldRG *cstor.RaidGroup) error {
+func (pOps *PoolOperations) validateNewBDCapacity(newRG, oldRG *cstor.RaidGroup) error {
```

Comment:
TODO: For me, in next PR's we can call `validateNewBDCapacity` function in `AreNewBDsValid` so that function name will full fill the need.